health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/presentation/preparing/PreparingExerciseScreen.kt

```diff
@@ -137,11 +133,9 @@ fun PreparingExerciseRoute(
 }
 
 private val grayscale = Paint().apply {
-    colorFilter = ColorFilter.colorMatrix(
-        ColorMatrix().apply {
-            setToSaturation(0f)
-        }
-    )
+    colorFilter = ColorFilter.colorMatrix(ColorMatrix().apply {
```

Comment:
where is this used?

Reply:
This is for ambient mode -- grayscale is defined here then used in the Modifier defined just below this, which sets the ambient mode color.

I didn't intentionally modify this, it got affected by the reformatting.